site/src/helpers/planner.ts

```diff
@@ -273,9 +273,9 @@ export const validatePlanner = (transferNames: string[], currentPlanData: Planne
       quarter.courses.forEach((course, courseIndex) => {
         if (!course.prerequisiteTree) return;
 
-        const { prerequisiteTree: prerequisite, corequisites: corequisite } = course;
+        const { prerequisiteTree: prerequisite } = course;
```

Comment:
Since we are only destructuring one thing now, this might read better as:

```suggestion
        const prerequisite = course.prerequisiteTree;
```